code/game/area/holodeck_areas.dm

```diff
@@ -0,0 +1,57 @@
+
+/area/holodeck
```

Comment:
not sure what to do with holodeck for now, since its also off station at times. Leaving it as it is for now.